### PR TITLE
fix(header): Styling corrected

### PR DESCRIPTION
**Pull request by Mardecera**
## Summary

- `Contact Us Button` : Changed its href to "/contact-us", added its active and focus.
- `Language Change Button` : Active and focus design was added.
- `Language Change Dropdown` : Its border radius was corrected, its drop-shadow was added and hover was added to its options.

### CODE REVIEW

**Review by CrhistianT7 — APPROVED** [review]
LGTM!!!

File: src/ui/Select/Select.styles.ts [hunk 1]
```diff
@@ -36,20 +59,19 @@ export const StyledSelectOptions = styled.div<IPosition>`
   ${(props) => (props.$position == 'left' ? 'left: 0;' : 'right: 0;')}
   min-width: 12.5rem;
   max-width: 21.875rem;
-  border: none;
   color: ${colors.black.main};
   border: 1px solid ${colors.tertiary.normal[200]};
-  border-radius: 1rem;
+  border-radius: 0.5rem;
+  box-shadow: rgba(0, 0, 0, 0.08) 0px 4px 12px;
```
Comment (GustavoUT22) [inline src/ui/Select/Select.styles.ts:65]: change rem instead of px

File: src/ui/Select/Select.styles.ts [hunk 2]
```diff
@@ -36,20 +59,19 @@ export const StyledSelectOptions = styled.div<IPosition>`
   ${(props) => (props.$position == 'left' ? 'left: 0;' : 'right: 0;')}
   min-width: 12.5rem;
   max-width: 21.875rem;
-  border: none;
   color: ${colors.black.main};
   border: 1px solid ${colors.tertiary.normal[200]};
```
Comment (GustavoUT22) [inline src/ui/Select/Select.styles.ts:63]: use rem

**Review by GustavoUT22 — APPROVED** [review]
Well done!

**Pull request merged**